app/build.gradle

```diff
@@ -29,6 +30,7 @@ android {
 
     buildFeatures{
         viewBinding = true
+        dataBinding = true
```

Comment:
Why do you have viewBinding and dataBinding?

Reply:
Back when I was researching MVVM I added dataBinding thinking I need it. Forgot to remove it later since I'm not using it.